jobs/FreeBSD-head-amd64-test_ltp/meta/run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+METADIR=/meta
+
+PATH=/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin:/usr/local/bin
+export PATH
+
+# Enable services needed by tests
+sysrc linux_enable="YES"
+service linux start
+
+chroot /compat/linux /opt/ltp/runltp -Q
+
+# XXX: Missing report generation
+echo $$ > test-report.xml
```

Comment:
Why do we use the process ID here instead of `$?` for the process return code. Does `runltp` return `0` anyway?

Reply:
Sorry, it was a typo.  As I said, this is completely untested yet.

Reply:
I see. Is it possible for us to have a quick check if all the tests are successful? It doesn't make much sense passing a value which is always 0 from VM. If we really want this, we can just ignore everything and let `build.sh` exits like it always does.

However, It's somehow important to have this checking mechanism. We need it to mark a build is failed or unstable, and then we can check the error. A always pass job will hide the error and it's usually too late when we notice them.

Reply:
Depends on how we handle the expected failures: I'd rather avoid marking them as such in LTP itself; I'd rather somehow just filter out the expected failures from the test results.  But that means reporting the overall status requires parsing the results, which is the piece we don't have yet.  Thus the temporary workaround of just reporting the exit code.

Reply:
I see. I can merge this and then modify the config in jenkins-job-builder (jjb) directory to create the job. I'm a little uncomfortable with an always pass test job. How's the LTP output and can we ask around if any volunteer want to work on this? I guess it could be grep'ing the case names and comparing with a expected failure list. Will this kind of script do the job?

Reply:
The format is somewhat weird; for example:
```
<<<test_start>>>
tag=abort01 stime=1574985777
cmdline="abort01"
contacts=""
analysis=exit
<<<test_output>>>
tst_test.c:1118: INFO: Timeout per run is 0h 05m 00s
abort01.c:57: PASS: abort() dumped core
abort01.c:60: PASS: abort() raised SIGIOT

Summary:
passed   2
failed   0
skipped  0
warnings 0
<<<execution_status>>>
initiation_status="ok"
duration=0 termination_type=exited termination_id=0 corefile=no
cutime=0 cstime=4
<<<test_end>>>
```
As I said, Alexander Leidinger (netchild@) offered to write the converter from this to JUnit, which is what I understand Jenkins uses natively.  Otherwise I'll try to do it.

Reply:
Got it. Perhaps we can ping Alexander and check the status.

The second format looks easier to parse and compare with the expected fail list.
Can ltp generate both formats in one run? If so, we can whip a quick script for the second one and show the first one in the console log. I really hope we can add the job along with a quick result check.

Reply:
I'll do that.

As for the format - true.  LTP always generates both, and by default the first one goes to the console, while the second goes to a file.

Reply:
That sounds very good. If you have the expected failure list, writing such script should be straightforward. What's the filename of the second one?